build.windows.ps1

```diff
@@ -0,0 +1,10 @@
+#! 
+# https://www.sitepoint.com/creating-your-first-gem/
+chef gem build $PSScriptRoot/kitchen-pester.gemspec
```

Comment:
Do you need to call `chef` here?  `gem build ...` will build the gem.

Reply:
Hmm, yeah. Should do. Assuming `gem` is on the path. Maybe safer to assume `chef` will be on the path, and it's essentially guaranteed to come with a working `gem` package?

Reply:
Yeah, given the context (TK), I'd rather stick with chef.
Also, this is a dirty helper that is convenient for testing, but far from critical.
I think it's mainly of use for noob like me trying to understand how to compile and install the gem.